data/11/mattheson/music-example1.xml

```diff
@@ -4,7 +4,7 @@
     <meiHead>
         <fileDesc>
             <titleStmt>
-                <title></title>
+                <title>Prob-Stück 11, Example 1</title>
```

Comment:
First shot on naming examples. Too brief?

Reply:
Looks good to me!